t04_refactored.py

```diff
@@ -137,6 +137,66 @@ def team_3_adv():
 def team_4_adv():
     pass
     # TODO Add your code here
+    # """
+    # https://docs.google.com/document/d/1BT_3NDURoZ_V-Ztx1OQcLPjn5krxuLOk8cLPmETCF2c/edit
+    # Kafui Gle
+    # Samundra Adhikari
+    # :return: none
+    # """
+from time import sleep
+
+delay = 1.0
+
+def main():
```

Comment:
You shouldn't be redefining a main() function. There is one already, at the bottom of this file. All of your code should be inside `team_4_adv()` function.

Reply:
main() function removed please. all our code has been place into the team_4_adv() function.